ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/aaa-settings-2.cfg

```
@@ -0,0 +1,76 @@
+!
+no enable password
+no aaa root
+!
+vlan internal order ascending range 1006 1199
+!
+transceiver qsfp default-mode 4x10G
+!
+service routing protocols model multi-agent
+!
+hostname aaa-settings-2
+!
+vlan 4092
+   name INBAND_MGMT
+!
+vrf instance INBAND_MGMT
+!
+vrf instance MGMT
+!
+management api http-commands
+   protocol https
+   no shutdown
+   !
+   vrf MGMT
+      no shutdown
+!
+radius-server host 11.10.10.11 key 7 071B245F5A
+radius-server host 11.10.10.1 vrf INBAND_MGMT key 7 071B245F5A
+radius-server host 11.10.10.2 vrf INBAND_MGMT key 7 071B245F5A
+radius-server host 11.10.10.3 vrf INBAND_MGMT key 7 071B245F5A
+radius-server host 11.10.10.4 vrf other_vrf key 7 071B245F5A
+!
+aaa group server radius RADIUS1
+   server 11.10.10.11 vrf default
+   server 11.10.10.1 vrf INBAND_MGMT
+   server 11.10.10.2 vrf INBAND_MGMT
+   server 11.10.10.3 vrf INBAND_MGMT
+   server 11.10.10.4 vrf other_vrf
+!
+aaa group server radius RADIUS2
+   server 11.10.10.1 vrf INBAND_MGMT
+!
+tacacs-server policy unknown-mandatory-attribute ignore
```

probably not needed multiple times ?
```suggestion
tacacs-server policy unknown-mandatory-attribute ignore
```

---

removed